src/base/module/record/Record.js

```diff
@@ -59,7 +59,20 @@ class Record {
     return this.sendRequest('GET', 'records', getRecordsRequest);
   }
 
-  getAllRecordsByQuery(app, query, fields, totalCount, offset, records) {
+  /**
+   * Get multi records more than default limitation number by query
+   * TODO: Parse to response model
+   * @param {Number} app
+   * @param {String} query
+   * @param {Array<String>} fields
+   * @param {Boolean} totalCount
+   * @return {Promise} Promise
+   */
+  getAllRecordsByQuery(app, query, fields, totalCount) {
+    return this.getAllRecordsByQueryRecursive(app, query, fields, totalCount, null, null);
+  }
+
+  getAllRecordsByQueryRecursive(app, query, fields, totalCount, offset, records) {
```

Comment:
I think this fucntion is internal and not intended for the developer to use directly, so maybe we should rename the function with an "_" prefix?
For example: _getAllRecordsByQueryRecursive

What do you think?

Reply:
@trinhhunganh 
＞I think this fucntion is internal and not intended for the developer to use directly
I think so. But any other internal functions (ex. addAllRecordsRecursive) were not implemented as so.
Is it OK? 
(I think it is better to update all internal functions...)

Reply:
@youtr-s 
Yes you are right, ideally all internal functions should be with '_' prefix.
So maybe I can just register a feedback. Please, do not modify your code at the moment.
Thank you!